preview-maker.js

```diff
@@ -68,9 +68,59 @@ const fill = (node, color) => {
         }
     }
 };
+const fillHSL = (node, hsl) => {
+    if (node.tagName === `stop`) {
+        node.setAttribute(`stop-color`, hsl);
+    } else {
+        node.setAttribute(`fill`, hsl);
+    }
 
+    if (node.childNodes) {
+        for (let child of Array.from(node.childNodes)) {
+            if (child.setAttribute) {
+                fillHSL(child, color, l);
+            }
+        }
+    }
+};
+function RGBToHSL(rgbArr){
```

Comment:
there's already a function in `@snejugal/color` to convert from RGB to HSL. can you replace usages of this function with `rgbToHsl` from `@snejugal/color`?